src/operators/localoperator.jl

```diff
@@ -57,3 +57,17 @@ function Base.repeat(O::LocalOperator, m::Int, n::Int)
     end
     return LocalOperator(lattice, terms...)
 end
+
+function Base.:+(O1::LocalOperator, O2::LocalOperator)
+    if O1.lattice != O2.lattice
+        throw("lattices should match")
+    end
+    terms = []
+    for (inds, operator) in O1.terms
+        push!(terms, inds => operator)
+    end
+    for (inds, operator) in O2.terms
+        push!(terms, inds => operator)
+    end
+    return LocalOperator(O1.lattice, terms...)
```

Comment:
I think the following might be a little easier:
```suggestion
    return LocalOperator(O1.lattice, O1.terms..., O2.terms...)
```
This being said, it might be worth it to check if there are duplicate coordinates, and in those cases add the operators together. This should improve efficiency.

Reply:
I don't see how this would work ? The LocalOperator constructor takes a lattice and an Ntuple ? Now you're giving it 3 arguments no ? Or is this some sort of julia magic that I don't understand ?

Reply:
I didn't do the suggested change. But I did implement the check for duplicate coordinates

Reply:
The constructor takes a lattice and a comma-separated-list of operators. Notice how the splatting (`...`) syntax just unpacks something into such a list, the following things are equivalent:
`LocalOperator(lattice, terms...) === LocalOperator(lattice, terms[1], terms[2], ..., terms[end])`
Thus, there is really no need to first make a vector that contains both, and then unpack them into such a list again. You might as well immediately unpack and concatenate.